### PR TITLE
Build Ubuntu 20.04 / Ruby 2.6 image

### PR DESCRIPTION
**Pull request by ags**
We were recently forced to upgrade our base ruby image from ubuntu 16.04 to 18.04 to get a new version of openssl. We'd been discussing moving to 20.04 beforehand, so now we're halfway there we'd like to finish that move.

In #13 we removed the postgres apt repository as the packages we needed were already available for 18.04. For 20.04 they're not, so this re-adds them according as per https://www.postgresql.org/download/linux/ubuntu/. We could do this only in the images derived from this one that actually use those packages, but since that's most of them it's a bit easier to do it once here.

See:

* https://github.com/conversation/docker-images/pull/13
* https://github.com/conversation/docs/pull/60

### CODE REVIEW

File: ruby-ubuntu/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,13 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 
 # This prevents questions being asked when package like tzdata are installed.
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update \
-      && apt-get install -y --no-install-recommends software-properties-common \
+      && apt-get install -y --no-install-recommends software-properties-common wget gpg-agent \
```
Comment (benmacleod) [inline ruby-ubuntu/Dockerfile:7]: Makes sense to me to add the prerequisites for installing postgres into our Ruby images. As you say, all or most of them will end up using postgres, and the prereqs in themselves won't add much bulk, but installing them here will take away some cognitive load elsewhere.

File: Makefile [hunk 2]
```diff
@@ -20,8 +20,8 @@ ruby:
 
 ruby-ubuntu:
 	cd $@; docker build -t $@ .
-	docker tag $@ theconversation/ruby:latest-ubuntu-bionic
-	docker tag $@ theconversation/ruby:2.6-ubuntu-bionic-202110010719
+	docker tag $@ theconversation/ruby:latest-ubuntu-focal
+	docker tag $@ theconversation/ruby:2.6-ubuntu-focal-202110011319
 
 
 sfdx:
```
Comment (benmacleod) [inline Makefile:27]: By the way, we won't end up using the `sfdx` image as didn't end up using `sfdx` to do Salesforce admin. So you can purge this config and the associated image(s) as you please.

**Review by benmacleod — APPROVED** [review]
Builds an Ubuntu 20.04 image to be used in our Ruby stacks in a natural progression from the existing image, the main difference being the addition of prerequisites for installing Postgres 👍🏻

**Comment by nullobject** [comment]
:+1:

File: Makefile (hunk 2 above)
Comment (ags) [reply Makefile:27]: I was wondering about that - I'll get rid of it, thanks 👍

**Pull request merged**